tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -206,12 +206,16 @@ var _ = Describe("odo devfile create command tests", func() {
 	})
 
 	Context("When executing odo create with --s2i flag", func() {
+		var newContext string
 		JustBeforeEach(func() {
-			newContext := path.Join(commonVar.Context, "newContext")
+			newContext = path.Join(commonVar.Context, "newContext")
 			devfilePath = filepath.Join(newContext, devfile)
 			helper.MakeDir(newContext)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", devfile), devfilePath)
 		})
+		AfterEach(func() {
```

Comment:
Nit: should this be `JustAfterEach`? Also, I think, we have a common `BeforeEach` & `AfterEach`.